core/lib/src/data/mod.rs

```diff
@@ -8,3 +8,4 @@ mod from_data;
 pub use self::data::Data;
 pub use self::data_stream::DataStream;
 pub use self::from_data::{FromData, FromDataSimple, Outcome, Transform, Transformed};
+crate use self::net_stream::NetStream;
```

Comment:
What makes use of this new import?

Reply:
It's used in `try_socket_stream_downcast`